src/popup_dictionary/web.py

```diff
@@ -173,7 +174,12 @@
         term = selection.toString().trim();
 
         // Return if selection empty or too short
-        if(term.length < 3){ return; }
+        var ignore =/%s/g
```

Comment:
It seems like this literal causes a syntax error, when %s is empty.

Reply:
Fixed